### PR TITLE
[#789] Rest API for training logs

### LINKED ISSUE #789: Add ability to get model training logs through EDI
In order to simplify model management and introspection procedures we have to add `legionctl log` command that will load or stream (as tail command) model container logs

### PR DESCRIPTION
**Pull request by mcdoker18**
* Currently operator supports retraining of a ModelTraining
* Added commitID field to status of ModelTraining CRD
* Added Rest API for training logs
* Training logs appears during `legionctl mt create` command
* Added command to view training logs

This closes #789

### CODE REVIEW

File: scripts/build-legion.Jenkinsfile [hunk 1]
```diff
@@ -182,10 +182,11 @@ pipeline {
                 stage("Build model builder and operator images") {
                     steps {
                         script {
-                            legion.buildLegionImage('k8s-model-builder', ".", "containers/operator/Dockerfile", "--target model-builder")
-                            legion.buildLegionImage('k8s-operator', ".", "containers/operator/Dockerfile", "--target operator")
-                            legion.buildLegionImage('k8s-edi', ".", "containers/operator/Dockerfile", "--target edi")
-                            legion.buildLegionImage('operator-tests', ".", "containers/operator/Dockerfile", "--target tests")
+                            legion.buildLegionImage('golang-dependencies', ".", "containers/operator/Dockerfile", "--target model-builder -t golang-dependencies:${Globals.buildVersion}")
```
Comment (kirillmakhonin) [inline scripts/build-legion.Jenkinsfile]: Not related to PR

File: legion/operator/pkg/webserver/routes/v1/model_trainning.go [hunk 2]
```diff
@@ -288,3 +297,53 @@ func (controller *ModelTrainingController) deleteMTByLabels(c *gin.Context) {
...
+		logFlushSize := viper.GetInt64(legion.LogFlushSize)
+
+		select {
+		case <-clientGone:
+			return
+		default:
+			_, err := io.CopyN(c.Writer, reader, logFlushSize)
+			if err != nil {
+				if err != io.EOF {
+					logMT.Error(err, "Copy the log stream")
```
Comment (kirillmakhonin) [inline legion/operator/pkg/webserver/routes/v1/model_trainning.go]: Not informative message

File: scripts/build-legion.Jenkinsfile (hunk 1 above)
Comment (mcdoker18) [reply scripts/build-legion.Jenkinsfile]: Fixed

File: legion/operator/pkg/webserver/routes/v1/model_trainning.go (hunk 2 above)
Comment (mcdoker18) [reply legion/operator/pkg/webserver/routes/v1/model_trainning.go]: Fixed

**Comment by mcdoker18** [comment]
bot, test app

**Review by kirillmakhonin — APPROVED** [review]

**Review by alexander-semenets — APPROVED** [review]

**Pull request merged**